src/external/views/management/management-view.tsx

```diff
@@ -10,11 +11,32 @@ import AppNumberInput from 'assets/bootstrap/input/number-input/number-input';
 import AppButton from 'assets/bootstrap/button/button';
 import SaveIcon from 'assets/images/icons/save/save';
 
+
 const AppManagementPage = () => {
     const httpClientService: HttpClientService = new HttpClientService();
 
     const navigate = useNavigate();
 
+   
+    const importExcel = (e) => {
+        const reader = new FileReader();
+        reader.readAsArrayBuffer(e.target.files[0]);
+        reader.onload = (e) =>{
+            const data = e.target.result;
+            const workbook = xlsx.read(data, {type : "array"});
+            const sheetName = workbook.SheetNames[0];
+            const sheet  = workbook.Sheets[sheetName];
+            const parseData = xlsx.utils.sheet_to_json(sheet);
+         httpClientService.post<Object,Object>( "/import/stock",parseData,{},true).then( (resp) => {
+
```

Comment:
WE should alert the user of its success and failure. However due to time I will approve